### ISSUE TITLE
Perfection Mode Hanging

### ISSUE THREAD

**Issue by jacket430** [hmxmilohax/rock-band-3-deluxe#699]
**Describe the bug**
Upon completing a song perfectly, the neck will disappear and the crowd will get excited, but nothing will happen beyond this point. The highway will disappear, and the characters will continue doing idle animations indefinitely. 

**To Reproduce**
1. Enable "Perfection Mode" modifier
2. Perfectly play a song, FC indicator pops up
3. Neck disappears
4. Soft lock

**Expected behavior**
Completion of the song, regular score saving + posting. 

**Screenshots**
Screenshot [here](https://media.discordapp.net/attachments/961395385014812672/1242266365662990336/image.png?ex=666053f5&is=665f0275&hm=00570a3f212cf43c40b4dbc42cc8a3c0589f445dc7db78f75c845d0ffcdce3c1&=&format=webp&quality=lossless&width=1193&height=671)

**DX Settings**
- `dx_settings.dta`: [dx_settings.txt](https://github.com/user-attachments/files/15571475/dx_settings.txt)
- `dx_modifiers.dta`: [dx_modifiers.txt](https://github.com/user-attachments/files/15571480/dx_modifiers.txt) (Perfection mode toggled off cause of the hanging)
- `dx_values.dta`: [dx_values.txt](https://github.com/user-attachments/files/15571485/dx_values.txt)

**Game Information**
 - Platform: RPCS3, Xbox 360* (RPCS3 is recent. Haven't tested 360 in a couple weeks, but was present last time I checked.)
 - Version: Not sure how to list nightly versions, but nightly push ebba7b6.
 - Song Count: 5,925

**Additional context**
No additional context.

**Comment by jnackmclain** [comment]
mind trying latest nightly and see if that helps? im thinking something with an improper restart was causing this

**Comment by jacket430** [comment]
Sorry for the late response, but just tested a song and failed it once just to be sure, and it appears to have been fixed. Great stuff, thanks fellas.

**Comment by jacket430** [comment]
Hey there,

I just ran into the game locking up again with perfection mode enabled, and I'm not sure it's related to the old custom being played, or the perfection mode hanging, but there are scores on ashcentral for the song so I assume it's working? [Here](https://www.youtube.com/watch?v=mXCY0etMMv8) is a video of the issue happening. 

The song is part of the C3 Legacy Pack ordered from Arby's about 3 days ago or so. I've played through other songs without issue. I also played Charlene (I'm Right Behind You) with perfection mode and all instruments mode enabled, and didn't have the issue.

Just thought I'd mention it on here since I'm pretty sure I was brainwashed to use the github.
